target/linux/ramips/dts/mt7621_wevo_w2914ns-v2.dtsi

```diff
@@ -11,19 +11,20 @@
 	};
 
 	keys {
-		compatible = "gpio-keys-polled";
-		poll-interval = <20>;
+		compatible = "gpio-keys";
 
 		reset {
 			label = "reset";
 			gpios = <&gpio0 29 GPIO_ACTIVE_LOW>;
 			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
```

Comment:
I thought debounce-interval is used for gpio-keys-polled?

Reply:
gpio-keys also uses debounce-interval so it will be okay.
https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/input/gpio-keys.txt#L25